index.js

```diff
@@ -100,13 +101,20 @@ module.exports = async (Component, opts = {}) => {
   const browser = await puppeteer.launch()
   const page = await browser.newPage()
   await page.goto(data)
+
+  let rect = {}
+  if (!width && !height) {
```

Comment:
Think I mentioned this on the other PR, but I think the CLI has default values for `width` and `height` – might make sense to add another explicit flag for this – maybe `--autosize`